### PR TITLE
Add docs for sysvar cache

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### What is changed, added or deleted? (Required)

This is the docs for pingcap/tidb#24359 which has not merged yet. It more accurately reflects the future-state of sysvars.

There is another PR for GA releases which describes how it previously worked better: #5562

It needs a "followup" to insert the version the change is effective from.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5563
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by morgo** [comment]
/cc @xhebox 

I can't read it :-)

File: system-variables.md [hunk 1]
```diff
@@ -25,8 +25,8 @@ SET  GLOBAL tidb_distsql_scan_concurrency = 10;
 
 > **注意：**
 >
-> - 在 TiDB 中，`GLOBAL` 变量的设置即使重启后也仍然有效。每隔 2 秒，其他 TiDB server 会获取到对变量设置的更改。详情见 [TiDB #14531](https://github.com/pingcap/tidb/issues/14531)。
-> - 此外，由于应用和连接器通常需要读 MySQL 变量，为了兼容这一需求，在 TiDB 中，部分 MySQL 5.7 的变量既可读取也可设置。例如，尽管 JDBC 连接器不依赖于查询缓存 (query cache) 的行为，但仍然可以读取和设置查询缓存。
+> - 当语句发出后，`SET GLOBAL` 会在 TiDB 服务器上立即执行。之后，通知会发送到所有 TiDB 服务器上以刷新其系统变量缓存，该操作会在后台立即开始。由于某些 TiDB 服务器可能会错过通知，系统变量缓存每 30 秒会自动刷新一次。这有助于确保所有服务器都以相同的配置运行。
```
Comment (xhebox) [inline system-variables.md:28]: > 当语句发出后，`SET GLOBAL` 会在 TiDB 服务器上立即执行。之后，通知会发送到所有 TiDB 服务器上以刷新其系统变量缓存

意思对了, 但是不太自然, 也许类似

> `SET GLOBAL` 语句在执行它的 TiDB 服务器上会立刻起效. 之后会通知所有TiDB 服务器...

**Review by xhebox — COMMENTED** [review]
Rest LGTM

File: system-variables.md (hunk 1 above)
Comment (Liuxiaozhen12) [reply system-variables.md]: 请问第一句改成这样可以如何？Thx！

> 在 TiDB 服务器上执行 `SET GLOBAL` 语句后，该更改会立即生效。之后会通知所有 TiDB 服务器刷新其系统变量缓存……

**Review by xhebox — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**